frontend/src/component/ui/Toast.scss

```diff
@@ -0,0 +1,33 @@
+@import '../styles/color';
+@import '../styles/corner';
+@import '../styles/font';
+@import '../styles/layouts';
+
+.toast:hover {
+  opacity: 1;
+  cursor: pointer;
+}
+
+.toast-message {
+  font-size: 14px;
+  text-align: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  @include default-font;
+}
+
+.toast {
+  z-index: 10000;
+  width: 80vw;
+  max-width: 300px;
+  color: #dedede;
+  background-color: $color-dark-gray;
+  position: fixed;
+  opacity: 0.9;
+  bottom: 5vh;
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
Let's add 
```
@mixin align-horizontal-center {
    left: 50%;
   transform: translateX(-50%);
}
```
to `frontend/src/component/styles/alignment` and use the mixin here to encourage reuse.